data/core/closetheloop.md

```diff
@@ -846,8 +846,8 @@
 * get_started_step4
     - action_greet_user
     - slot{"shown_privacy":true}
-* rasa_cost
-    - utter_rasa_cost
+* ask_faq_opensource_cost
+    - action_faqs
```

Comment:
@melindaloubser1 were there any more `rasa_cost` anywhere?

Reply:
Also replaced in step1_get_started.md ; no more that I could find

Reply:
👍